packages/components/app/styles/components/pagination-bar.scss

```diff
@@ -2,4 +2,127 @@
 // PAGINATIONBAR
 //
 
-// .hds-pagination-bar {}
+@use "../mixins/focus-ring" as *;
+
+.hds-pagination-bar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 36px;
+  color: var(--token-color-foreground-primary);
+  font-size: var(--token-typography-body-300-font-size);
+}
+
+// TODO: refactor!!! (Should not reference nested component classes)
+.hds-pagination-bar__page-size {
+  > * {
+    display: flex;
+    gap: 12px; // TODO: refactor
+    align-items: center;
+  }
+
+  // Issue: attributes are spread on nested select element, not on the component's parent wrapper...
+  .hds-form-label {
+    font-weight: inherit;
+    font-size: inherit;
+  }
+
+  .hds-form-select {
+    height: 28px;
+    margin-top: 0 !important;
+    padding-top: 0;
+    padding-right: calc(0px + 24px);
+    padding-bottom: 0;
+    padding-left: 8px;
+    font-size: inherit;
+    background-position: center right 5px;
+  }
+}
+
+.hds-pagination-bar__pagination {
+  display: flex;
+  align-self: stretch;
+}
+
+.hds-pagination-bar__page-list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+.hds-pagination__page {
+  display: flex;
+}
```

Comment:
I can see all states for `.hds-pagination__page--current`, but in `.hds-pagination__page` they seem to be missed.

Reply:
@cveigt I made some updates to refactor some things. Could you please check again and let me know your feedback? https://hds-components-git-hds-944-pagination-styling-hashicorp.vercel.app/components/pagination-bar#showcase